frontend/docsettings.lua

```diff
@@ -354,8 +358,8 @@ function DocSettings:removeSidecarDir(doc_path, sidecar_dir)
     end
 end
 
---- Updates sidecar info for file rename/copy/move/delete operations.
-function DocSettings:update(doc_path, new_doc_path, copy)
+--- Updates sdr location for file rename/copy/move/delete operations.
+function DocSettings:updateDocSettingsLocation(doc_path, new_doc_path, copy)
```

Comment:
`DocSettings:updateDocSettingsLocation(...)` - may be avoid the duplication ? what else's location could it be except itself?:
`DocSettings:updateLocation(...)`